src/utils/PrivateRoute.js

```diff
@@ -1,15 +1,30 @@
 import React, { useContext } from "react";
-import { Redirect, Route } from "react-router-dom";
+import { Redirect, Route, Link } from "react-router-dom";
 
 import { IsAuthContext } from "./AuthContext";
 
 function PrivateRoute({ children, ...restOfProps }) {
-  const isAuth = true
-  
-  return isAuth ? 
-    (<Route {...restOfProps}>{children}</Route>)
-       :
-    (<Redirect exact to="/" /> )
+  const { isLoggedIn } = useContext(IsAuthContext);
+  const { isAuth } = useContext(IsAuthContext);
```

Comment:
Can ben done in one line. You don't have to do import it separately as you're just destructering your `context` which is an object. 

```javascript 
const { isLoggedIn, isAuth } = useContext(IsAuthContext);
```